intranet/apps/eighth/models.py

```diff
@@ -1262,6 +1284,7 @@ def in_clear_absence_period(self):
         return self.scheduled_activity.block.in_clear_absence_period()
 
     def archive_remove_absence(self):
+        """If was_absent is True, sets was_absent=False, archived_was_absent=True, and saves both fields."""
```

Comment:
Probably could be reworded to become:
```suggestion
        """If user was absent, archives the absence."""
```